samples/DispatcherSample.Web/Startup.cs

```diff
@@ -39,10 +40,8 @@ public void ConfigureServices(IServiceCollection services)
                             response.ContentLength = payloadLength;
                             return response.Body.WriteAsync(_homePayload, 0, payloadLength);
                         },
-                        "/",
+                        RoutePatternFactory.Parse("/"),
                         new RouteValueDictionary(),
-                        new RouteValueDictionary(),
-                        new List<MatchProcessorReference>(),
```

Comment:
I'm open to discussion about what constructors we should have here. I'm also sensitive to the fact that we have like 9 parameters. We should try to make a decision here that will carry us through preview1.

Reply:
Will your average user ever want to create an endpoint themselves? If so then maybe one or two shorter ones? I don't have any strong opinions.

Reply:
I don't think that's likely in 2.2 the audience for now is frameworks/middlewarelikes.